doc/api/util.md

```diff
@@ -1024,6 +1024,227 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## Class: `util.MIMEType`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of [the MIMEType class](https://bmeck.github.io/node-proposal-mime-api/).
+
+In accordance with browser conventions, all properties of `MIMEType` objects
+are implemented as getters and setters on the class prototype, rather than as
+data properties on the object itself.
+
+A MIME string is a structured string containing multiple meaningful
+components. When parsed, a `MIMEType` object is returned containing
+properties for each of these components.
+
+### Constructor: `new MIMEType(input)`
+
+* `input` {string} The input MIME to parse
+
+Creates a new `MIMEType` object by parsing the `input`.
+
+```js
+const myMIME = new MIMEType('text/plain');
```

Comment:
Is this available as a new global? I assume not?

```suggestion
import { MIMEType } from 'util';

const myMIME = new MIMEType('text/plain');
```

(do we need the CJS equivalent too?)

Reply:
```suggestion
import { MIMEType } from 'node:util';

const myMIME = new MIMEType('text/plain');
```